server/odc-core/src/main/resources/i18n/BusinessMessages.properties

```diff
@@ -142,6 +142,7 @@ com.oceanbase.odc.LimitMetric.TABLE_NAME_LENGTH=table name length
 com.oceanbase.odc.LimitMetric.WORKSHEET_CHANGE_COUNT=number of changed worksheets
 com.oceanbase.odc.LimitMetric.WORKSHEET_SAME_LEVEL_COUNT=number of same level worksheets
```

Comment:
The number of worksheets at the same level

Reply:
okay, i'll make this change.